src/pykeen/stoppers/early_stopping.py

```diff
@@ -105,6 +174,11 @@ def __post_init__(self):
         #     raise ValueError(f'Invalid metric name: {self.metric}')
 
         self.remaining_patience = self.patience
```

Comment:
can you just refer to the remaining_patience  inside the `self._stopper` now?

Reply:
done in 16261a0